### PR TITLE
use trusted URL for AudioWorklet

### PR DESCRIPTION
**Pull request by glharper**

### CODE REVIEW

File: src/common.browser/WebsocketMessageAdapter.ts [hunk 1]
```diff
@@ -4,6 +4,9 @@
 // Node.JS specific web socket / browser support.
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```
Comment (BrianMouncer) [inline src/common.browser/WebsocketMessageAdapter.ts:7]: This PR is suppose to address some teams security best practices?
So I find it ironic that my biggest red flag are these eslint disable "unsafe blah" directives.

Can you please provide some justification for why these are needed, and why it is safe to do so in this case?

File: tsconfig.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
+    "target": "es6",
+    "module": "esnext",
```
Comment (BrianMouncer) [inline tsconfig.json:4]: is "esnext" stable, or will it change underneath our feet from build to build?

**Review by BrianMouncer — APPROVED** [review]
see comments.

**Review by dargilco — APPROVED** [review]

**Review by rhurey — APPROVED** [review]
:shipit:

File: tsconfig.json (hunk 2 above)
Comment (glharper) [reply tsconfig.json:4]: Understanding what the tsconfig module option does is non-trivial, but it appears have the module option as esnext should not change the output (while setting the target as esnext would).

File: src/common.browser/WebsocketMessageAdapter.ts (hunk 1 above)
Comment (glharper) [reply src/common.browser/WebsocketMessageAdapter.ts:7]: I'll look at what I can do to remove these disables, but the switch from es5 to es6 is what triggers the build errors in WebsocketMessageAdapter requiring these directives. es6 is stricter about enforcing eslint rules, apparently.

**Pull request merged**